js/annotations/navigationBindings.js

```diff
@@ -1361,6 +1361,16 @@ H.setOptions({
          * @apioption navigation.events.deselectButton
          */
 
+        /**
+         * Path where Highcharts will look for icons. Change this to use
+         * icons from a different server.
+         *
+         * @since 7.1.3
+         * @apioption @navigation.iconsURL
```

Comment:
`@apioption` must be last tag in doclet and option path must start witch letter.

Reply:
@TorsteinHonsi will fix it in master